lib/handlers_v5.js

```diff
@@ -341,3 +344,21 @@ function shutdown_request(request) {
 
     status_idle.call(this, request);
 }
+
+function is_complete_request(request) {
```

Comment:
For incomplete code, the transpile would undoubtedly fail. Which is fine in the case of coffeescript I think, because returning status `"unknown"` is at least not broken. However, esprima supports es6 syntax, so it would probably be better not to transpile in that case. How do you suggest I distinguish between babel and coffeescript? My first thought is to make the conditional:

```javascript
if (this.session.kernelInfoReply.language_info.implementation == "jp-coffeescript") {
    // ...
}
```

but that still seems far from ideal.